src/main/java/jewellery/inventory/mapper/OrganizationMapper.java

```diff
@@ -26,4 +31,48 @@ public OrganizationResponseDto toResponse(Organization organization) {
     organizationResponseDto.setNote(organization.getNote());
     return organizationResponseDto;
   }
+
+  public OrganizationMembersResponseDto toOrganizationMembersResponseDto(
+      Organization organization) {
+    OrganizationMembersResponseDto membersResponseDto = new OrganizationMembersResponseDto();
+    membersResponseDto.setMembers(toListUserInOrganizationResponseDto(organization));
+    membersResponseDto.setOrganization(toResponse(organization));
+    return membersResponseDto;
+  }
+
+  public OrganizationSingleMemberResponseDto toOrganizationSingleMemberResponseDto(
+      UserInOrganization userInOrganization) {
+    OrganizationSingleMemberResponseDto memberResponseDto =
+        new OrganizationSingleMemberResponseDto();
+    memberResponseDto.setMember(toListUserInOrganizationResponseDto(userInOrganization));
+    memberResponseDto.setOrganization(toResponse(userInOrganization.getOrganization()));
+    return memberResponseDto;
+  }
+
+  private List<UserInOrganizationResponseDto> toListUserInOrganizationResponseDto(
+      Organization organization) {
+    List<UserInOrganizationResponseDto> userResponseDtoList = new ArrayList<>();
+
+    organization
+        .getUsersInOrganization()
+        .forEach(
+            userInOrg -> {
+              UserInOrganizationResponseDto userResponseDto = new UserInOrganizationResponseDto();
+              userResponseDto.setUserId(userInOrg.getUser().getId());
+              userResponseDto.setOrganizationPermissions(userInOrg.getOrganizationPermission());
+              userResponseDtoList.add(userResponseDto);
+            });
+
+    return userResponseDtoList;
+  }
+
+  private UserInOrganizationResponseDto toListUserInOrganizationResponseDto(
```

Comment:
Both this method and the one on top are named the same. When you refactor one, it refactors the other as well. In this case you should unselect the other method when refactoring
<img width="1088" alt="Screenshot 2024-03-08 at 10 29 58" src="https://github.com/JewelleryManagement/jewellery-management-service/assets/33765600/5ea92282-8a23-41c6-bdd2-989ff8f0e2a9">